src/main/java/trackie/storage/Storage.java

```diff
@@ -0,0 +1,95 @@
+package trackie.storage;
+
+import trackie.tasks.Deadline;
+import trackie.tasks.Event;
+import trackie.tasks.Task;
+import trackie.tasks.Todo;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.FileReader;
+import java.io.PrintWriter;
+import java.util.Scanner;
+
+public class Storage {
+    private TaskList taskList;
+    private File fPtr;
```

Comment:
Perhaps `fPtr` could be written out as a full word, `filePointer` I'm guessing? So that it would be easier for other people reading your code to understand.